tests/phpunit/tests/media.php

```diff
@@ -5536,8 +5688,11 @@ static function ( $loading_attrs ) {
 			1
 		);
 
-		$this->assertSameSets(
-			array( 'fetchpriority' => 'high' ),
+		$this->assertSameSetsWithIndex(
+			array(
+				'decoding'      => 'async',
+				'fetchpriority' => 'high'
```

Comment:
```suggestion
				'fetchpriority' => 'high',
```

This will address PHPCS failure